README.md

```diff
@@ -180,23 +180,23 @@ depending on whether you want to use HTTPS or SSH. Finally, run the
 following:
 
 ```sh
-
 python3 -m pip install -r requirements.txt
 ```
 
 ## Usage
 
 To run your first example Scene, you can run the following commands:
 For users:
-1. Download the `example_scenes.py` file from [GitHub](https://github.com/ManimCommunity/manim/blob/master/example_scenes.py)
+
+1. Download the `example_scenes/basic.py` file from [GitHub](https://github.com/ManimCommunity/manim/blob/master/example_scenes/basic.py)
 2. Run the following command:
 ```sh
-manim example_scenes.py SquareToCircle -pl
+manim example_scenes/basic.py SquareToCircle -pl
 ```
 1. Run the following command:
 For developers:
 ```sh
-python3 manim.py example_scenes.py SquareToCircle -pl
+python3 manim.py example_scenes\basic.py SquareToCircle -pl
```

Comment:
This is wrong though? We don't have `manim.py` now.

Reply:
@kilacoda could you suggest the correct alternative? Or perhaps @XorUnison?

Reply:
It's the same for users and devs now.

Reply:
It should be `manim example_scenes.py SquareToCircle -pl`

That's after installing with `pip install -e .`

Reply:
Can we remove those lines then?

Reply:
Yes, you should edit them.